plasma_framework/test/src/exits/payment/PaymentExitGame.test.js

```diff
@@ -0,0 +1,141 @@
+const PaymentExitGame = artifacts.require('PaymentExitGame');
+const PlasmaFramework = artifacts.require('PlasmaFramework');
+const PriorityQueue = artifacts.require('PriorityQueue');
+const EthVault = artifacts.require('EthVault');
+const ExitId = artifacts.require('ExitIdWrapper');
+
+const { BN, constants } = require('openzeppelin-test-helpers');
+const { expect } = require('chai');
+
+const { MerkleTree } = require('../../../helpers/merkle.js');
+const { PaymentTransactionOutput, PaymentTransaction } = require('../../../helpers/transaction.js');
+const Testlang = require('../../../helpers/testlang.js');
+
+contract('PaymentExitGame - End to End Tests', ([_, alice, bob]) => {
+    const MIN_EXIT_PERIOD = 60 * 60 * 24 * 7; // 1 week
+    const STANDARD_EXIT_BOND = 31415926535; // wei
+    const ETH = constants.ZERO_ADDRESS;
+    const DEPOSIT_VALUE = 1000000;
+
+    const aliceDeposits = async () => {
+        const depositBlockNum = (await this.framework.nextDepositBlock()).toNumber();
+        this.depositUtxoPos = Testlang.buildUtxoPos(depositBlockNum, 0, 0);
+        this.depositTx = Testlang.deposit(DEPOSIT_VALUE, alice);
+        this.merkleTreeForDepositTx = new MerkleTree([this.depositTx], 16);
+        this.merkleProofForDepositTx = this.merkleTreeForDepositTx.getInclusionProof(this.depositTx);
+
+        await this.ethVault.deposit(this.depositTx, { from: alice, value: DEPOSIT_VALUE });
+    };
+
+    const aliceTransferToBob = async () => {
+        const tranferTxBlockNum = (await this.framework.nextChildBlock()).toNumber();
+        this.transferUtxoPos = Testlang.buildUtxoPos(tranferTxBlockNum, 0, 0);
+
+        const output = new PaymentTransactionOutput(1000, bob, ETH);
+
+        // should be output_id instead of utxoPos?
```

Comment:
According to MVP1 spec  transaction input that should be output_id, but I don't think we want to do it like this for payment transactions.

Reply:
I update this comment with the tracking issue: https://github.com/omisego/research/issues/93.
use utxo_pos for now as it does not require function to build output_id > <